recipes-kernel/linux/linux-nilrt.inc

```diff
@@ -1,6 +1,29 @@
-inherit kernel
+LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
+LIC_FILES_CHKSUM_xilinx-zynq = "file://COPYING;md5=d7810fab7487fb0aad327b76f1be7cd7"
```

Comment:
We can remove these `xilinx-zynq` components.

Reply:
Do we not support zynq targets anymore?

Reply:
Your right, I was looking at an older hash of the hardknott branch. Did you want me to go a head and strip all zynq stuff in recipes-kernel as part of this PR or should I make another PR if that would make things cleaner? Or did we want a tech debt item to strip all zynq stuff from meta-nilrt?

Reply:
Your choice as to whether you do it now, or make a TD item to to it later. It doesn't matter if it is a part of this PR or not.

Reply:
azdo: [2123329](https://dev.azure.com/ni/DevCentral/_workitems/edit/2123329) has been created to capture the remove of zynq and arm from the meta-nilrt layer since there are multiple areas that will need to be updated.